src/pages/en/legal/cozy-privacy-policy.mdx

```diff
@@ -96,15 +75,15 @@ that we have collected to another organization, or directly to you, under certai
 your personal data, under certain conditions.
 
 If you would like to exercise those rights, contact us at [privacy@quiltmc.org](mailto:privacy@quiltmc.org).
-We may ask you to verify your identity before processing your request, using a service such as [Aperture](https://aperture.starchild.systems).
-We will respond to your request within 30 days as required by law.
+We may ask you to verify your identity before processing your request.
+We will respond to your request within one month as required by law.
```

Comment:
Article 12.3 of the GDPR:
> The controller shall provide information on action taken on a request under [Articles 15](https://gdpr-info.eu/art-15-gdpr/) to [22](https://gdpr-info.eu/art-22-gdpr/) to the data subject without undue delay and in any event within one month of receipt of the request.

As vague as it might be, the law says one month, which can be relevant in some jurisdictions due to the month of February.

Reply:
After further review, it seems I have misread this diff. Please disregard this comment.